lib/bundler/templates/Executable

```diff
@@ -8,8 +8,15 @@
 # this file is here to facilitate running it.
 #
 
+def bundle_stub?(path)
```

Comment:
defining the method like this will leak it to the global scope. Can you also add a test for that this is fixing? Thanks!